### ISSUE TITLE
Display upto 4 entitles per row?

### ISSUE THREAD

**Issue by aptonline** [nervetattoo/banner-card#3]
At the moment there seems to be a maximum of 3 entities that can be displayed per row before another row is created below it, would it be possible to increase this to 4 entities per row?

I first thought it was the length of the entity output or name length that dictated this but even with 'small' entities the maximum seems to be 3.

**Comment by nervetattoo** [comment]
They all fill 1 / 3 width now. I felt like that was just the right size to make it look good in just about all instances. I don't like providing foot guns via configuration, but I agree that it makes sense to be able to tweak this.

My current plan is to make it into a definable grid:

```yaml
row_size: 4
entities:
  - entity: light.bar
     size: 1
  - entity: light.bar
     name: A really really really long name
     size: 3
  - entity: media_player.living_room
     size: full
```

**Comment by nervetattoo** [comment]
Fixed in #1f0ddd4 and release in 0.4.0 🥂
